### ISSUE TITLE
Kill specific processes before game

### ISSUE THREAD

**Issue by juancarlospaco** [bung87/gamode#2]
Somehow allow a customizable list of processes to kill before running a game,
to kill custom installed programs automatically before running a game, maybe it can be like a `config.cfg` ? 
(programs like Telegram, Discord, Flameshot, bots, torrents, etc).

**Comment by bung87** [comment]
good idea, could provide a select box let user choose multiple program and save to config file.

**Comment by juancarlospaco** [comment]
Another cool idea can be to kill Cortana (Search Indexer).
